### DISCUSSION TITLE
Create newsletter

### DISCUSSION THREAD

**Discussion by acoluzzi** [latina-in-tech/latina-in-tech.github.io#30] [Ideas]
### Description of the new feature
We want to add a newsletter so to update community members about new events and stuff.

- Add a panel in the website 'Newsletter' which will ask the user for their email and register it to a 3rd party provider newsletter
- We can use newsletter provider, such as [Mailchimp](https://mailchimp.com/it/?currency=EUR), which has a free plan of 1K monthly emails (use Latina In Tech google account -> ask admin to sign up)

### Tech Details
- use the following regex for email: `^([a-z][a-z0-9]{1,13}[_.-]?[a-z0-9]{1,13})@((([0-9]{1,3})([.]{1})([0-9]{1,3})([.]{1})([0-9]{1,3})([.]{1})([0-9]{1,3}))|([a-z0-9]{1,10}[-]?[a-z0-9]{1,10}[.]?[a-z0-9-]{1,10}[.]{1}[a-z]{2,4}))$` 
which is safe as per [regex DOS](https://devina.io/redos-checker)
- Try to hide auth token in the source code, perhaps using github secrets

**Comment by Undermaken** [comment]
@coluzziandrea I'm playing with https://convertkit.com/ 
it seems to match our needs

<img src="https://github.com/latina-in-tech/latina-in-tech.github.io/assets/822471/17596085-2f60-49c4-97e5-79cb216f5258" height="500"/>

I haven't done any tests yet but you should be able to design your form to allow users to sign up and then integrate into your website as an embedded element.

In this way, we don't need to write any code.

Thoughts? Could we have any block by adopting it?

**Comment by acoluzzi** [comment]
We can give it a try, @Undermaken!

**Comment by acoluzzi** [comment]
I'm just wondering if it exists some provider that will allow us to use our HTML, but I couldn't find any until now

**Comment by Undermaken** [comment]
I worked with https://github.com/Faire/mjml-react but I discovered that in email/html world there are no standards and a lot of edge cases. 

Btw if we keep the email simple, that's a way

**Comment by acoluzzi** [comment]
@Undermaken should we convert this issue to a discussion until we decide a way?

**Comment by Undermaken** [comment]
Absolutely, yes!
I'm not the maximum expert about that and any contribution, it is gold

**Comment by acoluzzi** [comment]
Here we are, let's try a bunch of ways and share here
